internal/models/cluster/dataprotection/spec.go

```diff
@@ -0,0 +1,40 @@
+package dataprotectionmodels
+
+import "github.com/go-openapi/swag"
+
+// VmwareTanzuManageV1alpha1ClusterDataprotectionSpec The spec collects all the options for installing backup and restore solution into a Kubernetes cluster.
+//
+// swagger:model vmware.tanzu.manage.v1alpha1.cluster.dataprotection.Spec
+type VmwareTanzuManageV1alpha1ClusterDataprotectionSpec struct {
+
+	// List of Backup Locations to install in the cluster.
+	BackupLocationNames []string `json:"backupLocationNames"`
```

Comment:
This field is technically not deprecated in the api but is not recommended to be used. Is it ok to drop this now or does it need to be deprecated in the api first?

Reply:
It is not a problem to get rid of it as long as we also don't let you update this through TF.
Currently it is configured part of the schema as a field to be filled.

Reply:
Would prefer to have it removed since we don't want to encourage users to populate this. @eastaga wdyt?

Reply:
Yes it should be removed from TF.

Reply:
Done